### ISSUE TITLE
no results

### ISSUE THREAD

**Issue by Edrinrmc** [danielhuici/Wallamonitor#10]
I followed all the steps, but the request never returns results; it’s always 0, even though I know my spec should return results
this was my specs:
{
    "search_query": "WH1000XM4",
    "latitude": "xxxxxx",
    "longitude": "xxxxxxx",
    "max_distance":"300",
    "condition": "all",
    "min_price": "20",
    "max_price": "200",
    "title_exclude" : [],
    "description_exclude": [],
    "title_must_include" : [],
    "title_first_word_exclude":[] ,
    "description_must_include" : []
  }
Could something have changed in the API?

**Comment by danielhuici** [comment]
Check your parameter `max_distance`. Distance must be specified in meters not kilometers!

**Comment by Edrinrmc** [comment]
Checked and the result is always the same.

**Comment by danielhuici** [comment]
Depending on your selected max_distante, and also if other params are so restrictive for your item, it may take some time until new articles appear

**Comment by patternseek** [comment]
Should the monitor notify all existing matching listings when it first runs or does it make a list of what's available when it first runs and then just notifies when new items are added? Because I'm seeing something similar to Edinrmc, however loose the parameters I enter I never get any results.

**Comment by patternseek** [comment]
Sorry ignore my message. I had a look at the code and I see that it does the latter and only notifies on articles posted after the worker starts.

**Comment by danielhuici** [comment]
Yes, it's like that. The tool first lists existing items as it doesn't make any sense to notify existing products in the platform
